resources/views/competitions/create.blade.php

```diff
@@ -4,4 +4,34 @@
 
     <h1>Create A New Competition</h1>
 
+    <div class='container__block'>
+        <div class='wrapper'>
+            {!! Form::open(['route' => 'competitions.store']) !!}
+                <div class="form-item">
+                    {{ Form::label('campaign_id', 'Campaign ID:', ['class' => 'field-label']) }}
+                    {{ Form::text('campaign_id', NULL, ['class' => 'text-field']) }}
+                </div>
+
+                <div class="form-item">
+                    {{ Form::label('campaign_run_id', 'Run ID:', ['class' => 'field-label']) }}
+                    {{ Form::text('campaign_run_id', NULL, ['class' => 'text-field']) }}
+                </div>
+
+                <div class="form-item">
+                    {{ Form::label('start_date', 'Start date:', ['class' => 'field-label']) }}
+                    {{ Form::input('date', 'start_date', NULL, ['class' => 'text-field']) }}
+                </div>
+
+                <div class="form-item">
+                    {{ Form::label('end_date', 'End date:', ['class' => 'field-label']) }}
+                    {{ Form::input('date', 'end_date', NULL, ['class' => 'text-field']) }}
+                </div>
+
+                <div class="form-item">
+                    {{ Form::submit('Submit', ['class' => 'button']) }}
+                </div>
+            {!! Form::close() !!}
```

Comment:
Just a note @deadlybutter we can leave this using the `Form` facade stuff for now, but I'll likely change it for the front-end later. I've started moving away from using the `Form` facade for most of the interior input elements cause it's more versatile to just use the actual regular html. The only place where the `Form` facade is useful if for the `Form::open()`